packages/base-cli/src/io-fs-to-fs.ts

```diff
@@ -1,4 +1,4 @@
-import fsp from 'fs/promises';
+import { promises as fsp } from 'fs';
```

Comment:
I often just do this

```suggestion
import fs from 'fs/promises';
```

But feel free to leave it like this if you want to be specific

Reply:
I mostly use the specific naming. This is more a reminder for myself that I imported the promises API further down in the code. :)